Backpack-SwiftUI/Dialog/Classes/DialogContainerViewModifier.swift

```diff
@@ -30,57 +30,40 @@ struct DialogContainerViewModifier<DialogContent: View>: ViewModifier {
     @ViewBuilder let dialogContent: DialogContent
     let onTouchOutside: (() -> Void)?
     
-    @State private var controller: UIViewController?
-    
     func body(content: Content) -> some View {
-        content
-            .onChange(of: isPresented) { _ in
-                if isPresented {
-                    showDialogWithContent {
-                        ZStack {
-                            Color(.scrimColor)
-                                .ignoresSafeArea()
-                                .onTapGesture { onTouchOutside?() }
-                            HStack {
-                                Spacer(minLength: .lg)
-                                dialogContent
-                                    .frame(maxWidth: 400)
-                                Spacer(minLength: .lg)
-                            }
-                        }
+        ZStack {
+            content
+            if isPresented {
+                ZStack {
```

Comment:
If we already have our content in a ZSgtack, why do we need a second one here?

Reply:
This is because we need to group the content and apply certain modifiers to the group. It so happens that the content of the dialog is also a zstack but that's a ''''''detail'''''' of the dialog content.